clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -7,12 +7,20 @@
   {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 
 # Test Variales
-  {{$PODS_PER_NODE := .PODS_PER_NODE}}
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 1}}
   {{$DEPLOYMENT_TEMPLATE_PATH := .DEPLOYMENT_TEMPLATE_PATH }}
-  {{$VOLUMES_PER_POD := .VOLUMES_PER_POD}}
+  {{$VOLUMES_PER_POD := DefaultParam .VOLUMES_PER_POD 1}}
   {{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+  # Set this to false if metrics data is not needed.
+  {{$GATHER_METRICS := DefaultParam .GATHER_METRICS true}}
```

Comment:
I still believe this should be removed due to reasons discussed in the other thread (in particular api-call latencies SLO is measured behind this param and it should be measured).

Reply:
This depends on why you run the test. If it is to measure performance, then yes. But I was also using it to trigger a race condition (https://github.com/kubernetes-csi/external-provisioner/issues/507) and then metrics gathering is just unnecessary overhead. In particular in a large cluster it can take time as it walks through various pods.

Reply:
I think there shouldn't be an option for that, but ok - let's not block this PR on it.